linters/sort-package-json/plugin.yaml

```diff
@@ -0,0 +1,20 @@
+version: 0.1
+lint:
+  files:
+    - name: package-json
+      filenames: [package.json]
```

Comment:
Defining files in plugins has other side effects, primarily since we don't support namespacing file types (so any plugins that define file types must therefore be globally namespaced). This isn't really a problem, and we already do this already, but we've been defining all file types at the top level plugin.yaml just for visibility and ease of access for the future. Can you move this there?

Reply:
Alternatively, there is some benefit to having some proximity to it's main use case (as in the case of the circle ci linter, which does the same thing. Ultimately your call, but wanted to highlight the choice that we had been making up to that point.

Reply:
Leaving it here for now for the same reason as circleci, but if we ever have a 2nd user we should definitely move it to the root